kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeOutlinePanel.js

```diff
@@ -114,38 +114,40 @@ define(['jquery', 'base/js/namespace', 'kbwidget', 'kbaseNarrativeControlPanel']
          * @returns {OutlineItem[]} - Array of outline items
          */
         getOutlineItems: function () {
-            const outline_items = [];
+            const outlineItems = [];
             Jupyter.notebook.get_cells().forEach((cell) => {
                 // Find any headers in the cell
-                let inner_headers = cell.element[0].querySelectorAll('h1, h2, h3');
+                let innerHeaders = cell.element[0].querySelectorAll('h1, h2, h3');
                 // Differentiate between cell types for creating items
-                if (cell.cell_type === 'markdown' && inner_headers.length !== 0) {
+                if (cell.cell_type === 'markdown' && innerHeaders.length !== 0) {
                     // Create header items for markdown cell
                     if (!this.options.showAllHeaders) {
-                        inner_headers = [inner_headers[0]];
+                        innerHeaders = [innerHeaders[0]];
                     }
-                    const md_items = Array.from(inner_headers).map((h, i) => ({
+                    const mdItems = Array.from(innerHeaders).map((h, i) => ({
                         title: h.innerText,
                         depth: parseInt(h.nodeName[1]),
                         element: i === 0 ? cell.element[0] : h,
-                        in_view: this.cellInView(cell.element[0]),
+                        inView: DisplayUtil.verticalInViewport(cell.element[0]),
                         selected: cell.selected,
                         icon: i === 0 ? cell.getIcon() : '',
+                        scrollTo: () => Jupyter.narrative.scrollToCell(cell, true),
```

Comment:
Items now have a `scrollTo` property which is a function that calls `Jupyter.narrative.scrollToCell` on the appropriate cell